src/Collector/ThemeCollector.php

```diff
@@ -29,15 +33,6 @@ final class ThemeCollector extends DataCollector
 
     private ThemeHierarchyProviderInterface $themeHierarchyProvider;
 
-    /**
-     * @var array
-     *
-     * @psalm-var array{used_theme: ?ThemeInterface, used_themes: ThemeInterface[], themes: ThemeInterface[]}
-     *
-     * @psalm-suppress NonInvariantDocblockPropertyType
-     */
-    protected $data;
```

Comment:
This variable is defined by DataCollector itself. With beginning of Symfony 7 it has a Union Type here: https://github.com/symfony/symfony/blob/1a16ebc32598faada074e0af12a6a698d2964a5e/src/Symfony/Component/HttpKernel/DataCollector/DataCollector.php#L31

For BC reasons best is to move this variable to PHPDoc definition. So we can support Symfony 7 and previous versions of the parent collector class.